expertise/run.py

```diff
@@ -18,23 +17,37 @@
         submissions_dataset = SubmissionsDataset(submissions_file=Path(config['dataset']['directory']).joinpath('submissions.jsonl'))
 
     if config['model'] == 'bm25':
+        from .models import bm25
         bm25Model = bm25.Model(
             use_title=config['model_params'].get('use_title'),
             use_abstract=config['model_params'].get('use_abstract'),
-            workers=config['model_params'].get('workers')
+            workers=config['model_params'].get('workers'),
+            sparse_value=config['model_params'].get('sparse_value')
         )
         bm25Model.set_archives_dataset(archives_dataset)
         bm25Model.set_submissions_dataset(submissions_dataset)
-        bm25Model.all_scores(Path(config['model_params']['scores_path']).joinpath(config['name'] + '.csv'))
+
+        if not config['model_params'].get('skip_bm25', False):
+            bm25Model.all_scores(
+                preliminary_scores_path=Path(config['model_params']['scores_path']).joinpath('preliminary_scores.pkl'),
+                scores_path=Path(config['model_params']['scores_path']).joinpath(config['name'] + '.csv')
+            )
+        if config['model_params'].get('sparse_value'):
```

Comment:
Am I correct in saying that this generates both `all_scores` and `sparse_scores`? If so, why do we need both scores?

Reply:
You're right, it's OK to only have one